core/trino-parser/src/main/java/io/trino/sql/SqlFormatter.java

```diff
@@ -320,28 +309,23 @@ protected Void visitQuerySpecification(QuerySpecification node, Integer indent)
         {
             process(node.getSelect(), indent);
 
-            if (node.getFrom().isPresent()) {
+            node.getFrom().ifPresent(from -> {
                 append(indent, "FROM");
                 builder.append('\n');
                 append(indent, "  ");
-                process(node.getFrom().get(), indent);
-            }
+                process(from, indent);
+            });
 
             builder.append('\n');
 
-            if (node.getWhere().isPresent()) {
-                append(indent, "WHERE " + formatExpression(node.getWhere().get()))
-                        .append('\n');
-            }
+            node.getWhere().ifPresent(where ->
+                    append(indent, "WHERE " + formatExpression(where)).append('\n'));
```

Comment:
nit
```suggestion
            node.getWhere().ifPresent(where -> append(indent, "WHERE " + formatExpression(where))
                    .append('\n'));
```

for consistency with other two-append calls below.